### PR TITLE
updated link in example pattern file

### PR DESCRIPTION
**Pull request by srajaw**
*Issue #, if available:*

*Description of changes: Only updated one link in example pattern file for a existing pattern and added one new pattern. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apigw-ec2/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Invoke cross account Step Function -> AWS Lambda Function
```
Comment (mavi888) [inline apigw-ec2/README.md:1]: I think the title doesnt match the pattern. Please fix

File: apigw-ec2/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/apigw-ec2
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+* [Create two AWS accounts for cross account setup](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have, create them and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
```
Comment (mavi888) [inline apigw-ec2/README.md:14]: I dont think this matches the architecture proposed. Please fix

File: apigw-ec2/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+1. Change directory to the pattern directory:
+    ```
+    cd serverless-patterns/apigw-ec2
+    ```
+1. Delete all created resources in same account.
+    ```
+    sam delete --stack-name <stackname>
+    ```
+----
+Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (mavi888) [inline apigw-ec2/README.md:78]: fix to 2023

File: apigw-ec2/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+## Requirements
+* [Create two AWS accounts for cross account setup](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have, create them and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) install and configure one profiles with credentials for the accounts as below:
+    ```
+    [default]
+    ```
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [SAM](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) installed
```
Comment (mavi888) [inline apigw-ec2/README.md:21]: Also mention in the preresites if they need their own vpc and what other things they need to have configured in their account to be able to deploy this.

File: apigw-ec2/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    #Enter a existing security group Id in which you want to deploy the resources. Make sure that security group allows port 80 traffic from the VPC CIDR range or from 0.0.0.0/0. This is required to allow traffic from NLB and install NGINX files successfully on EC2 machine
+    - Enter a SecurityGroupIDsForNLBAndEC2:
+
+    #Enter a Route Table Ids used by the above entered private subnet. This is required to create a VPC Endpoint which is responsible for fetching NGINX files for EC2.  
+    - Enter a RouteTableId:
+
+## Testing
+
+1. You can now invoke the REST API by executing a GET request via the below CURL command or from the browser. Copy the output from the Outputs section after deployment.
```
Comment (mavi888) [inline apigw-ec2/README.md:65]: Which curl command? Can you please provide it?

**Review by mavi888 — COMMENTED** [review]
Thanks for submitting this PR. There are some issues that need to be fixed before I can merge it. 

Cheers

**Comment by srajaw** [comment]
@mavi888 , thanks for pointing out these changes. I have corrected them. 

Also, CURL command will be present in the Outputs Section after deployment. Thanks!

**Comment by mavi888** [comment]
Thanks for submitting your pattern.
You can find it in here : https://serverlessland.com/patterns/apigw-ec2

Cheers

**Pull request merged**